src/style.css

```diff
@@ -1,3 +1,65 @@
+html {
+  --head-marg: 50px;
+  --max-header-wight: 1200px;
+  --min-header-wight: 1024px;
+  --special-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  align-items: center;
+  margin: 0 var(--head-marg);
+}
+
+.headerContent {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+li {
   margin: 0;
+  display: flex;
+  align-items: center;
+  align-content: center;
+  list-style: none;
```

Comment:
Styling of `li` elements with a tag selector can lead to unwanted side effects in other parts of the application. It's better to use a class selector to style specific list items to avoid such issues.